spec/fixtures/files/pvc.yml

```diff
@@ -0,0 +1,11 @@
+:apiVersion: v1
```

Comment:
also ":" prefixes here, not sure if this is valid yaml.

Reply:
as above

Reply:
thanks